news/_posts/core-weekly/2021-08-02-coreweekly-30-2021.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 30 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-08-02
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 26th of July to Sunday 1st of August 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Release patch 1.7.7.6 is [close to being delivered](https://github.com/PrestaShop/PrestaShop/issues/25202#issuecomment-889844880) while Release candidate 1.7.8.0 is also [close to the exit door](https://github.com/PrestaShop/PrestaShop/issues/23010#issuecomment-890881293).
+
+We are also preparing a new format for the incoming [Public OSS Demo](https://build.prestashop.com/news/upcoming-demo-7-2021/).
+
+You might have noticed some heavy work being done [on the developer documentation](https://github.com/PrestaShop/docs/pull/1095). We will soon have [to maintain multiple versions of the documentation](https://github.com/PrestaShop/docs/issues/1090), this requires changes in how we control the content. Two new repositories: [Devdocs theme](https://github.com/PrestaShop/ps-docs-theme) and [Devdocs sources](https://github.com/PrestaShop/devdocs-site) have been created.
```

Comment:
```suggestion
You might have noticed some heavy work being done [on the developer documentation](https://github.com/PrestaShop/docs/pull/1095). We will soon have [to maintain multiple versions of the documentation](https://github.com/PrestaShop/docs/issues/1090), this requires changes in how we build the site: the content will stay in the Docs repository, while the site itself has been moved to an [independent DevDocs site repository](https://github.com/PrestaShop/devdocs-site). Also, the [DevDocs' theme](https://github.com/PrestaShop/ps-docs-theme) has also been moved to an independent repository so it can be reused in the near future.
```